test/elm/interval/data.js

```diff
@@ -106623,84 +106643,108 @@ module.exports['Collapse'] = {
                } ]
             }
          }, {
-            "localId" : "398",
+            "localId" : "391",
+            "name" : "NullCollapse",
+            "context" : "Patient",
+            "accessLevel" : "Public",
+            "annotation" : [ {
+               "type" : "Annotation",
+               "s" : {
+                  "r" : "391",
+                  "s" : [ {
+                     "value" : [ "","define ","NullCollapse",": " ]
+                  }, {
+                     "r" : "390",
+                     "s" : [ {
+                        "r" : "389",
+                        "value" : [ "collapse ","null" ]
+                     } ]
+                  } ]
+               }
+            } ],
+            "expression" : {
+               "localId" : "390",
+               "type" : "Null"
+            }
```

Comment:
Due to the translation error, the ELM is actually just a literal `Null` -- so it's not actually testing the `Collapse` code. I think we can fix this in the test CQL via a cast.